sample/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
```

Comment:
Travis currently has build tools 23 set up in .travis.yml and I see it in other build.gradle files. I'd recommend bumping it everywhere for tools and compile version and I bet it'll resolve the travis issue